pysqldb3/tests/test_csv_to_table.py

```diff
@@ -341,6 +415,79 @@ def test_csv_to_table_basic(self):
+
+        df.to_csv(helpers.DIR + "\\test_dates.csv", index=False)
+
+        # csv_to_table
+        sql.query('drop table if exists {}.{}'.format(sql_schema, create_table_name))
+
+        fp = helpers.DIR + "\\test_dates.csv"
+        sql.csv_to_table(input_file=fp, table=create_table_name, schema=sql_schema)
+
+        # Check to see if table is in database
+        assert sql.table_exists(table=create_table_name, schema=sql_schema)
+        db_df = sql.dfquery("select * from {}.{}".format(sql_schema, create_table_name))
+
+        # Get csv df via pd.read_csv
+        csv_df = pd.read_csv(fp)
+        csv_df.columns = [c.replace(' ', '_').lower() for c in list(csv_df.columns)]  # TODO: look into this difference
+        csv_df = csv_df.astype({'false_date': 'string'})
+
+        # Assert df equality, excluding dtypes and columns
+        pd.testing.assert_frame_equal(db_df, csv_df, check_dtype=False)
+        # assert that dates have been set to varchar
+        for (name, typ) in sql.get_table_columns(create_table_name, schema=sql_schema):
+            if 'date' in name.lower():
+                print(name, typ)
+                assert typ == 'varchar'
+        # Cleanup
+        sql.drop_table(schema=sql_schema, table=create_table_name)
+        os.remove(fp)
+
+    def test_csv_to_table_basic_auto_date_w_override(self):
+        data = {'id': {0: 1, 1: 2, 2: 3, 3: 4, 4: 5},
+                'year': {0: 1981, 1: 2009, 2: 1954, 3: 1993, 4: 1973},
+                'sale date': {0: '1/1/2015', 1: '2/25/2012', 2: '7/9/2018', 3: '12/2/2021', 4: '11/13/1995'},
+                'false Date': {0: 0, 1: 0, 2: 0, 3: 0, 4: 0}
+                }
+        df = pd.DataFrame(data)
+
+        df.to_csv(helpers.DIR + "\\test_dates.csv", index=False)
+
+        # csv_to_table
+        sql.query('drop table if exists {}.{}'.format(sql_schema, create_table_name))
+
+        fp = helpers.DIR + "\\test_dates.csv"
+        sql.csv_to_table(input_file=fp, table=create_table_name, schema=sql_schema,
+                        column_type_overrides={'false Date': 'int'})
+
+        # Check to see if table is in database
+        assert sql.table_exists(table=create_table_name, schema=sql_schema)
+        db_df = sql.dfquery("select * from {}.{}".format(sql_schema, create_table_name))
+
+        # Get csv df via pd.read_csv
+        csv_df = pd.read_csv(fp)
+        csv_df.columns = [c.replace(' ', '_').lower() for c in list(csv_df.columns)]  # TODO: look into this difference
+        csv_df = csv_df.astype({'false_date': 'int'})
+
+        # Assert df equality, excluding dtypes and columns
+        pd.testing.assert_frame_equal(db_df, csv_df, check_dtype=False)
+        # assert that dates have been set to varchar
+        for (name, typ) in sql.get_table_columns(create_table_name, schema=sql_schema):
+            if 'date' in name.lower() and name.lower() != 'false_date':
+                print(name, typ)
+                assert typ == 'varchar'
```

Comment:
is this same as typ == 'character varying'?
And I think this should be good if there's no AssertionError raised.